src/Blorc.OpenIdConnect/wwwroot/oidc-client-interop.js

```diff
@@ -89,8 +89,8 @@
                     return false;
                 }
 
-                var self = this;
-                return new Promise((resolve, reject) => {
+                let self = this;
+                return new Promise((resolve, _reject) => {
                     self.userManager.signinRedirect();
                     resolve(true);
```

Comment:
Is the difference with signout (return vs no return) correct?

Reply:
removed return keyword